docs/nextflow-create-docker.md

```diff
@@ -56,7 +57,7 @@ Before you proceed with this command in your Dockerfile, you want to make sure y
 
 First, open your Docker Desktop application (just to be sure Docker is running).
 
-Next, open your terminal. Paste the fetch tag command you copied from Quay. If it's working, you will see language that it is pulling (see below). When it's complete (and successfully pulled), there will be a line that says `Status: Downloaded <image>` (see yellow highlight below). If you see this, you know that all the steps necessary to pull your image work. If you don't see this, reach out to us on Slack.
+Next, open your terminal. Paste the fetch tag command you copied from Quay. If it's working, you will see language that it is pulling (see below). When it's complete (and successfully pulled), there will be a line that says `Status: Downloaded <image>` (see yellow highlight below). If you see this, you know that all the steps necessary to pull your image work. If you don't see this, reach out to support at brhsupport@datacommons.io or on [Slack](https://docs.google.com/forms/d/e/1FAIpQLSczyhhOXeCK9FdVtpQpelOHYnRj1EAq1rwwnm9q6cPAe5a7ug/viewform).
```

Comment:
I think for the restricted testing group (even if we expand it, it's still by invitation) we want people to reach out to us on the Nextflow external testing slack channel that Karen and Jordan are on. If they email, it could take a while before it's seen. I want to be respectful of our testers time. I will rewrite this when we plan to make this a publicly-available feature.